script/lazytox.py

```diff
@@ -1,238 +0,0 @@
-#!/usr/bin/env python3
```

Comment:
I'd like to keep lazytox as I use it sometimes.

Reply:
Okay! Why, though – as in, what does it do better than running `pre-commit` with certain flags (i.e. could it be made a wrapper for `pre-commit`)? Just trying to clean up TMTOWDI bits and pieces in the repo :)

Reply:
Mainly cause `pre-commit` doesn't lint our tests.